platform/linux-generic/odp_pool.c

```diff
@@ -1220,6 +1220,7 @@ int odp_pool_info(odp_pool_t pool_hdl, odp_pool_info_t *info)
 
 	} else if (pool->type_2 == ODP_POOL_DMA_COMPL) {
 		info->dma_pool_param.num        = pool->params.buf.num;
+		info->dma_pool_param.uarea_size = pool->params.buf.uarea_size;
```

Comment:
Not strictly subject of this PR, but `odp_pool_print()` and `odp_pool_print_all()` implementations could be improved to properly recognize DMA pools.